src/opnsense/scripts/interfaces/traffic_top.py

```diff
@@ -94,15 +95,32 @@ async def request_ittr(self, addresses):
         dnsResolver.timeout = 2
         tasks = []
         for address in addresses:
-            tasks.append(dnsResolver.resolve_address(address))
+            try:
+                if ":" in address:
+                    ipv6_obj = ipaddress.IPv6Address(address)
+                    reverse_ip = '.'.join(reversed(ipv6_obj.exploded.replace(":", ""))) + ".ip6.arpa"
+                else:
+                    reverse_ip = '.'.join(reversed(address.split("."))) + ".in-addr.arpa"
+
+                tasks.append(dnsResolver.resolve(reverse_ip, "PTR"))
+            except Exception as e:
+                return
```

Comment:
continue? the next address might be a valid one according to the loop

Reply:
Good catch. I fixed this in my next commit.